janusgraph-cql/src/test/java/org/janusgraph/graphdb/cql/CQLGraphTest.java

```diff
@@ -138,4 +139,10 @@ public void testCustomConfigUsedByTx() {
                 .get(WRITE_CONSISTENCY));
         tx.rollback();
     }
+
+    @RepeatedIfExceptionsTest(repeats = 3)
+    @Override
+    public void simpleLogTest() throws InterruptedException{
+        super.simpleLogTest();
+    }
```

Comment:
Could be great if you could explain why you Overwrided the `super` test to make it Flacky for CQL

Reply:
https://github.com/JanusGraph/janusgraph/issues/1457

Reply:
It was never marked as flakytest. I often did just restart the failing tests on the TravisCI instead of marking Thema as flaky.